microsoft-edge/extensions-chromium/store-policies/developer-policies.md

```diff
@@ -5,147 +5,221 @@ author: MSEdgeTeam
 
-If you make any changes to your extension beyond the described functionality, any changes to code must be compliant with the [Microsoft Edge content security policy](csp.md#relaxing-the-default-policy).  For example, your extension should not download a remote script and subsequently run that script in a manner that is not consistent with the described functionality.
+Extensions that function as crypto wallets can only be published by verified company accounts.
 
-#### 1.2.2 Unwanted and Malicious Software
+In general, your extension must not jeopardize or compromise user security, or the security or functionality of the device, system, or related systems.
 
-Your extension must not contain or enable malware as defined by the Microsoft criteria for [Unwanted and Malicious Software](/windows/security/threat-protection/intelligence/criteria).
 
-#### 1.2.3 Dependency on other software
+###### 1.2.1 Content security policies
 
-Your extension may depend on non-integrated software (such as another product, module, or service) to deliver the primary functionality, provided you disclose the dependency in the description.
+If you make any changes to your extension beyond the described functionality, any changes to code must be compliant with the Content Security Policy for Microsoft Edge extensions; see [Content Security Policy (CSP)](./csp.md).  For example, your extension should not download a remote script and subsequently run that script in a manner that is not consistent with the described functionality.
 
-#### 1.2.4 Extensions Update
 
-Unless otherwise permitted by Microsoft, your extensions must be updated only through Microsoft Edge Add-ons.
+###### 1.2.2 Unwanted and malicious software
 
-### 1.3 Product is Testable
+Your extension must not contain or enable malware that may harm the operation of the network, servers, and infrastructure of Microsoft or any third parties. Spyware, malicious scripts, and phishing scams are also prohibited.
 
-Your extension must be testable.  If it is not possible to test your extension for any reason, including, but not limited to, the items below, your extension may fail this requirement.
+Your extension must comply with the Microsoft criteria for Unwanted and Malicious Software, listed in [How Microsoft identifies malware and potentially unwanted applications](/windows/security/threat-protection/intelligence/criteria).
 
-#### 1.3.1 User Credentials
 
-If your extension requires login credentials, provide a working demo account using the `Notes for certification` field.
+###### 1.2.3 Dependency on other software
 
-#### 1.3.2 Availability of services
+Your extension may depend on non-integrated software (such as another product, module, or service) to deliver the primary functionality. However, this information should be clearly disclosed in the description.
+
+
+###### 1.2.4 Extension updates
+
+Unless otherwise permitted by Microsoft, your extension must be updated only through Microsoft Edge Add-ons.
```

Comment:
```suggestion
Unless otherwise permitted by Microsoft, your extension must be updated only through the Microsoft Edge Add-ons store.
```
@Bhuvana-Priya 
Existing sentence doesn't make sense at end: add-ons must be updated through add-ons.

Reply:
@Bhuvana-Priya - is this what's meant?
Unless otherwise permitted by Microsoft, your extension must be updated only through Partner Center and the Edge Add-ons store.

Reply:
applied the change